test/TestUtilities/Test.Utility/MockServer.cs

```diff
@@ -43,20 +46,14 @@ public class MockServer : IDisposable
         /// <summary>
         /// Initializes an instance of MockServer.
         /// </summary>
-        public MockServer()
+        /// <param name="authenticationSchemes">The optional <see cref="AuthenticationSchemes" /> to use.</param>
+        public MockServer(AuthenticationSchemes authenticationSchemes = AuthenticationSchemes.Anonymous)
```

Comment:
This param name should probably be singular: authenticationScheme

Reply:
In this case `AuthenticationSchemes` can be a bitwise combination and support multiple authentication schemes at the same time so it should be plural.